src/Providers/ZeroRPCContextProvider.php

```diff
@@ -46,7 +46,7 @@ public function register()
                 $middleware->beforeSendRequest()
             );
 
-            if ($this->app->getProvider('Barryvdh\Debugbar\ServiceProvider')) {
+            if ($this->app->has('Barryvdh\Debugbar\LaravelDebugbar')) {
```

Comment:
I'm not sure since which version that laravel has `has` method, but I think it doesn't have it in 5.1